src/cli/domain/package-components.js

```diff
@@ -12,12 +12,10 @@ var getUnixUtcTimestamp = require('../../utils/get-unix-utc-timestamp');
 var validator = require('../../registry/domain/validators');
 
 module.exports = function(){
-  return function(componentPath, minify, callback){
```

Comment:
I changed this signature to accept an options parameter instead of adding another one